docs/authorisation.md

```diff
@@ -28,11 +28,54 @@ Where possible, IAM Roles should be used.
 
 ## IP addresses
 
+The purpose of IP addresses in this context, are to help determine origin. Thus, grant trust and offer privileges as a result.  
+
 IP addresses in and of themselves do not constitute authentication but may be considered a minor authentication *indicator* when combined with other authentication and authorisation techniques.
 
 For example, traffic originating from a perceived known IP address/range does not automatically mean it is the perceived user\(s\) however it could be used as an indicator to *reduce* \(not eliminate\) how often [MFA](multi-factor-authentication-mfa-guide.md) is requested *within* an existing session.
 
-H/T [https://medium.com/@joelgsamuel/ip-address-access-control-lists-are-not-as-great-as-you-think-they-are-4176b7d68f20](https://medium.com/@joelgsamuel/ip-address-access-control-lists-are-not-as-great-as-you-think-they-are-4176b7d68f20)
+Where applicable, maintain a single source of truth. Automatically apply confirmed changes when trying to identify the IP address ranges. There is an importance of advance change notification as well as the consequences if not up kept.  
```

Comment:
Maybe use:



### Maintain IP address lists

Where applicable, maintain a single source of truth with meaningful labels to describe each IP address range.

The use of infrastructure as code to both store and apply IP address lists will help reduce errors, and aid with change management.

Where practical, periodically check the IP addresses with the team responsible for those IP addresses, to cater for upcoming changes in IP spacing or change of use/scope.

Reply:
Many thanks for this. In the recognition, would you like a link to your LinkedIn or Twitter page to be included?

Reply:
> Many thanks for this. In the recognition, would you like a link to your LinkedIn or Twitter page to be included?

joelgsamuel.com would be great if thats OK? That has all of my links, otherwise my Twitter (https://twitter.com/joelgsamuel) please :) Thank you very much.

Reply:
I tried to visit the web address and I couldn't connect. Is the above the correct web address?

Reply:
> I tried to visit the web address and I couldn't connect. Is the above the correct web address please?

Yes, https://twitter.com/joelgsamuel is correct and also https://medium.com/@joelgsamuel/ip-address-access-control-lists-are-not-as-great-as-you-think-they-are-4176b7d68f20 is correct (you can use https://joelgsamuel.medium.com/ip-address-access-control-lists-are-not-as-great-as-you-think-they-are-4176b7d68f20 which is Medium's new URL format, but to be honest its all the same!)

Reply:
Hello Joel, unfortunately we aren't able to link to your personal comms. Do you have a LinkedIn account you would like us to link to at all?

Reply:
> Hello Joel, unfortunately we aren't able to link to your personal comms. Do you have a LinkedIn account you would like us to link to at all?

http://linkedin.com/in/jgsamuel/ should do the trick :)